src/content/getusermedia/source/index.html

```diff
@@ -43,10 +43,18 @@
 
     <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Select audio &amp; video sources</span></h1>
 
-    <p>Get available audio and video sources from <code>MediaStream.getSources()</code> then set the source for <code>getUserMedia()</code> using a <code>sourceId</code> constraint.</p>
+    <p>Get available audio, video sources and audio output devices* from <code>mediaDevices.enumerateDevices()</code> then set the source for <code>getUserMedia()</code> using a <code>sourceId</code> constraint.</p>
+
+    <p>*Experimental support in Chrome 45.0.2441.x or later using command line flag "--enable-blink-features=EnumerateDevices,AudioOutputDevices". Use the <code>setSinkID()</code> method on the video element and provide a MediaStream and a sinkId (sourceId for where <code>sourceInfos.kind === 'audiooutput'</code>) as arguments.</p>
```

Comment:
Maybe 'Enable experimental support in Chrome 45.0.2441.x or later by selecting <a href="chrome://flags/#enable-experimental-web-platform-features">Enable experimental Web Platform features</a> or by using command line flag ...'